tools/challenge-helper-scripts/tsconfig.json

```diff
@@ -4,7 +4,7 @@
 
     /* Basic Options */
     // "incremental": true,                         /* Enable incremental compilation */
-    "target": "ES2020",                                /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019', 'ES2020', 'ES2021', or 'ESNEXT'. */
+    "target": "ES2020" /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019', 'ES2020', 'ES2021', or 'ESNEXT'. */,
```

Comment:
Is this valid?

Reply:
Yep, prettier's just moved the commas around.  I'll remove the comments themselves in another PR, since it's easy enough to simply look them up.

Reply:
Ah. I did not see them at the end there.

Reply:
Nope, prettier has not lived up to it's name here.

Reply:
I went ahead and made #43550 to clean this file up, since this PR needs a rebase anyway 🙂